src/stack.js

```diff
@@ -1,5 +1,31 @@
 'use strict'
 
 export default class Stack {
```

Comment:
Initializing `this._topIndex` to `-1` feels unintuitive to me because indices typically start at `0`. 

Instead of starting at `-1` and using a preincrement to increment `this._topIndex` in your `push()` implementation, you can alternatively start at `0` and use a postincrement to increment `this._topIndex`.

This change would simplify your other method implementations.

For example:

```javascript

length() {
  return this._topIndex
}

peek() {
  return this._topIndex ? null : this._storage[ this._topIndex ]
}
```

...etc.